components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/common/service_constants.h

```diff
@@ -26,6 +26,10 @@ constexpr char kBraveVpnWireguardServiceUnnstallSwitchName[] = "uninstall";
 // Load wireguard binaries and connect to vpn using passed config.
 constexpr char kBraveVpnWireguardServiceConnectSwitchName[] = "connect";
```

Comment:
same as `common/win`, this is an incorrect usage of `common` in chromium

Reply:
this is separate executable and common code inside it

Reply:
Common to what? I think you are using "common" here with a different meaning and we should not be doing that because it's confusing in the context of chromium code. If it's common to the browser process code and exe then it should go in `brave_vpn/common/wireguard` or something

Reply:
and for instance here https://github.com/brave/brave-core/pull/18960/files#diff-032ee8426b314ef9f20a5b383cd195294523101c75b533685d14d2665e8ce71fR1 having `common` makes no sense because that code is only used for status_tray

Reply:
sure, reworking of `components/brave_vpn/browser/connection/common`  and `components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/common`  is out of scope for this PR and affects many files across browser process, ikev2 helper process and wireguard service preocess, I will take it next as separate PR in follow up https://github.com/brave/brave-browser/issues/31217

This one was added here `components/brave_vpn/browser/connection/wireguard/win/brave_vpn_wireguard_service/status_tray/common` and I renamed it to utils.

Reply:
Moving a file is not "out of scope". This is a very simple change to correct a problem with a file you made a change to in this PR.

Reply:
@bridiver done here https://github.com/brave/brave-core/pull/18960/commits/eab1cddf23d4ec17f4728999a810967efbba6809